www/static/css-src/_artwork.scss

```diff
@@ -0,0 +1,34 @@
+.iconset {
+  padding: 100px 0;
+}
+
+.button {
+    display: block;
+    background: url(/static/img/sprite.png) no-repeat;
```

Comment:
Please prefix the URI with `{{ site.baseurl }}`, and put the URL in double-quotation marks.